test/cli/compiler/test-migrations.js

```diff
@@ -0,0 +1,29 @@
+const test = require("tape");
+const path = require("path");
+const process = require("process");
+const testUtils = require("../../../bin/tools/utils");
+const fsp = require("fs").promises;
+const qx = path.resolve(__dirname, "..", "..", "..", "bin", "source", "qx");
```

Comment:
const qx = testUtils.getCompiler()

Reply:
Strangely, that breaks the tests, because getCompiler() returns the built version of the compiler instead of the source version. However, that means that there's something wrong, since both versions should produce the same output. Let me check.